parsel/selector.py

```diff
@@ -427,10 +521,12 @@ def get(self) -> str:
         Serialize and return the matched nodes in a single string.
         Percent encoded content is unquoted.
         """
+        if self.type in ("text", "json"):
+            return self.root
```

Comment:
I wonder if code would be simpler if we remove `type="text"`, and use `type="json"` for these cases as well. `if self.type == "text"` would be replaced with `self.type == "json" and insinstance(self.root, str)`. Not 100% sure about it, but there are signs that we can do it.

Reply:
https://github.com/scrapy/parsel/pull/181#discussion_r1007810927